kvmagent/kvmagent/plugins/shared_block_plugin.py

```
@@ -731,6 +743,188 @@ def get_retry_times_for_checking_vg_lockspace():
+
+            sanlock_hostname = "%s-%s-%s" % (cmd.vgUuid[:8], cmd.hostUuid[:8], linux.get_hostname()[:20])
+            lvm.modify_sanlock_config("our_host_name", sanlock_hostname)
+            shell.call("sed -i 's/.*rotate .*/rotate 10/g' /etc/logrotate.d/sanlock", exception=False)
+            shell.call("sed -i 's/.*size .*/size 20M/g' /etc/logrotate.d/sanlock", exception=False)
+
+        config_lvm(cmd.hostId, cmd.enableLvmetad)
+
+        lvm.start_lock_service(cmd.ioTimeout)
+
+        # 1 注册 生成一个新的vg 和 选中的 lun wwid
+        # 选中存储上vg 和 lun wwid，指定了新的vgUuid
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)